content/applications/hr/time_off/allocations.rst

```diff
@@ -93,16 +116,16 @@ view. Both buttons open a new allocation request form.
 
 Enter the following information on the new allocation request form:
 
-- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+- :guilabel:`Time Off Type`: Select the type of time off being requested for the allocation from the
   drop-down menu. After a selection is made, the title updates with the time off type.
-- :guilabel:`Validity Period`: using the calendar selector, select a start and end date the
+- :guilabel:`Validity Period`: Using the calendar selector, select a start and end date the
   allocation is valid for. By default, the current date populates the first field, and :guilabel:`No
   Limit` populates the second field. Adjust both of these dates, if desired.
-- :guilabel:`Allocation`: enter the amount of time being requested in this field. The format is
+- :guilabel:`Allocation`: Enter the amount of time being requested in this field. The format is
   presented in either :guilabel:`Days` or :guilabel:`Hours`, depending on how the :guilabel:`Time
   Off Type` is configured. Once this field is populated, the name of the allocation request is
   updated to include the amount of time being requested.
-- :guilabel:`Add a reason...`: enter a description for the allocation request. This should include
+- :guilabel:`Add a reason...`: Enter a description for the allocation request. This should include
```

Comment:
```suggestion
- :guilabel:`Add a reason...`: Enter a description for the allocation request in this field. This should include
```
added "...in this field" to ensure it was a complete sentence, to fall in line with your 'capitalization' being used.